src/python/turicreate/data_structures/serialization.py

```diff
@@ -0,0 +1,51 @@
+
+
+# SFrames require a disk-backed file or directory to work with.  This directory 
+# has to be present to allow for serialization or deserialization. 
+__serialization_directory = None 
+
+
+def enable_sframe_serialization(serialization_directory):
+    """
+    Enables pickling of sframes through the use of a user-set directory to 
+    store the objects.  This directory must be present and set through 
+    this method for deserialization to work, but may be a different directory.
+    """
```

Comment:
I think you should add more details here about the limitations. i.e. the user can't just copy pickle files to a new computer and have things work.